connectors/init.go

```diff
@@ -32,14 +32,17 @@ var configs = map[string]Config{}
 
 func Register[TConfig Config](f func(cfg TConfig) (Connector, error)) {
```

Comment:
let's better add a func - `RegisterAlias(typ string, aliases ...string)`, it's will be cleaner.